src/bridge/cockpitpackages.c

```diff
@@ -927,17 +928,34 @@ package_content (CockpitPackages *packages,
   gchar *chosen = NULL;
   gboolean globbing;
   gboolean gzipped;
+  gboolean allow_gzipped = FALSE;
   const gchar *type;
   gchar *policy;
+  gint i;
```

Comment:
Tiny style nitpick: it's better to declare such helper variables in the scope where they are needed, in this case `for (int i = 0;  ...`

Reply:
Sure thing. Done.